src/core_plugins/kibana/public/context/query_parameters/actions.js

```diff
@@ -43,7 +46,13 @@ export function QueryParameterActionsProvider() {
     )
   );
 
+  const addTermsFilter = (state) => (field, values, operation) => {
+    const { queryParameters: { indexPatternId } } = state;
+    filterManager.add(field, values, operation, indexPatternId);
```

Comment:
So while trying to add this I noticed that bumping the field popularity when adding a filter never worked in 5.x, because `indexPattern.popularizeField()` was always passed an incorrect argument for filtering operations. It only worked for column adding/removing. Do we still want to add it to the context view and fix it in other places?